charts/values.yaml

```diff
@@ -40,6 +40,9 @@ controller:
   # indicate we create the watcher jobs in user namespace, these allow users
   # to view the terraform output
   enableWatchers: true
+  # enableTerraformVersions indicates configurations are permitted to override
+  # the terraform version in their spec.
+  enableTerraformVersions: true
```

Comment:
I think this flag might have been clearer as enableTFVersionOverrides, but as long as the description and docs cover then it's fine